scripts/core/interactive-article-actions-panel/subcomponents/destination-select.tsx

```diff
@@ -59,11 +59,11 @@ export class DestinationSelect extends React.PureComponent<IProps> {
                         })()}
                         items={destinations}
                         onChange={(val) => {
-                            if (val.id === PERSONAL_SPACE) {
+                            if (val != null && val.id === PERSONAL_SPACE) {
```

Comment:
how about you do `if (val == null) { return; }` at the top and keep the remaining code unchanged? Antoher issue is that if there was a value, and user removed the value and closed the panel - it would be expected the value NOT to be there if he reopens the pane.

Reply:
Yes good point on one hand about the `if` handling. Another thing is that originally in sd-develop you can't have an empty value there. Check out how it works on that instance and you'll see.

Reply:
if UI doesn't allow empty value then all good.